src/core/client/stream/tabs/comments/components/Indent.tsx

```diff
@@ -20,6 +20,7 @@ const Indent: StatelessComponent<IndentProps> = props => {
           [styles.level3]: props.level === 3,
           [styles.level4]: props.level === 4,
           [styles.level5]: props.level === 5,
+          [styles.level6]: props.level === 6,
```

Comment:
Would it not be better to write some kind of getter function instead of this??

Reply:
I'm doing it this way, so that it is typechecked. Do you know a better way that does preserve the type checking?

Reply:
Ok I refactored it, should be better now :-)